felix/fv/utils/utils.go

```diff
@@ -53,6 +54,10 @@ type EnvConfig struct {
 var Config EnvConfig
 
 func init() {
+	envErr := godotenv.Load("../.env")
+	if envErr != nil {
+		log.Warningf("Error loading .env file! Err = %s", envErr)
```

Comment:
The .env file isn't mandatory right?  In which case I think this is more like a Debug log